doc/rtd/explanation/format.rst

```diff
@@ -78,6 +76,80 @@ Content found to be gzip compressed will be uncompressed.
 The uncompressed data will then be used as if it were not compressed.
 This is typically useful because user data is limited to ~16384 [#]_ bytes.
 
+``include`` file
+================
+
+This content is an :file:`include` file.
+
+The file contains a list of URLs, one per line. Each of the URLs will be read
+and their content will be passed through this same set of rules, i.e., the
+content read from the URL can be gzipped, MIME multi-part, or plain text. If
+an error occurs reading a file the remaining files will not be read.
+
+Begins with: ``#include``.
+
+Example
+-------
+
+Include 3 files:
+:file:`file/path/A`, :file:`file/path/B` and :file:`file/path/C`.
+
+.. code-block::
+
+   #include
+   file/path/A
+   file/path/B
+   file/path/C
+
+
+``cloud-boothook``
+==================
+
+One line ``#cloud-boothook`` header and then executable payload.
+
+This is run very early on the boot process, during the
+:ref:`Network boot stage<boot-Network>`, even before ``cc_bootcmd``.
+
+This can be used when something has to be configured very early on boot,
+potentially on every boot, with less convenience as ``cc_bootcmd`` but more
+flexibility.
+
+.. note::
+   Boothooks are execute on every boot.
+   The environment variable ``INSTANCE_ID`` will be set to the current instance
+   ID. ``INSTANCE_ID`` can be used to implement a `once-per-instance` type of
+   functionality.
```

Comment:
maybe add a final little sentence saying as demonstrated below in [link to "Example of once-per-instance script" section]